### PR TITLE
Data tab and tech doc tidying and bug fixing

### PR DESCRIPTION
**Pull request by jvillacampa**
I have simplified a few bits of code, tidied and commented sections. I have reorganized the filters, so they are ordered from biggest to smallest geography. Main improvement was fixing the bug that crashed the app when an user tried to download data without having selected an indicator/domain or profile

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -14,8 +14,11 @@ function(input, output, session) {
...
-    p(tags$h4("Welcome to the ScotPHO Profiles Tool"), tags$br(),tags$br(),tags$div(h5("This interactive tool provides access to a range of public health related indicators at different"),
-                                                                                    h5("geographies including NHS boards, council areas and health and social care partnerships.")),
+    p(tags$h4("Welcome to the ScotPHO Profiles Tool"), tags$br(),tags$br(),
+      tags$div(h5("This interactive tool provides access to a range of public 
+                  health related indicators at different"),
+               h5("geographies including NHS boards, council areas and health and 
+                  social care partnerships.")),
       style = "color:0E3E5D; font-size:20px"),
     br(),
     p(tags$h5("Find out more about how to get the most out of the tool")),
```
Comment (vaelliott) [inline server.R]: I suggest a change to the text to make it snappier -

"Learn more about the tool or get started"

File: server.R [hunk 2]
```diff
@@ -35,18 +38,27 @@ showModal(welcome_modal)
...
     fluidRow(
-      column(5, offset=1,tags$h5("You can select visualisations via the tabs at the top of the page")),
+      column(5, offset=1,tags$h5("You can select visualisations via the tabs at 
+                                 the top of the page")),
       column(3, br(), img(src="tabs_select.png", height=43, width=400))),
     fluidRow(
-      column(6, br(), br(), br(), br(), tags$a(img(src="landing_select.png", height=150, width=250, align="right"))),
-      column(4, br(), br(), br(),br(),  br(), br(), br(), br(), tags$h5("Or select to view a visualisation using the option buttons shown on the Home Page"))),
+      column(6, br(), br(), br(), br(), 
+             tags$a(img(src="landing_select.png", height=150, width=250, align="right"))),
```
Comment (vaelliott) [inline server.R]: technically this image shows the Map selection but this is no longer an option - would be good to change the image here but can wait - i'll put on trello

File: server.R [hunk 3]
```diff
@@ -2052,44 +2256,22 @@ showModal(welcome_modal)
...
     content = function(file) {
       write.csv(indicator_csv(),
                 file, row.names=FALSE) } 
   )
   
-  ########################
-  #Re-open Modal
-  ########################
+  ########################.
+  #Re-open Modal ----
```
Comment (vaelliott) [inline server.R]: I'm not sure where this is called? is this solving the issue of reopening the welcome modal at any point?

Comment (jvillacampa) [reply server.R]: out

**Pull request merged**